projects/plugins/social/src/class-meta-tags.php

```diff
@@ -279,23 +279,26 @@ public function render_tags( $post = null ) {
 			$tags['og:description'] = $this->get_description( $excerpt );
 		}
 
-		$image = $this->get_featured_image();
+		$featured_image = $this->get_featured_image();
+		// Add SIG image if enabled.
+		$images = apply_filters( 'jetpack_images_pre_get_images', array( $featured_image ), $data->ID );
```

Comment:
I'm not 100% sure if we should use this filter to add the image. It runs on `Jetpack_PostImages::get_image()`, which is also used by other things (such as Related Posts).

What do you think about hooking into the `jetpack_open_graph_tags` filter instead? We could try to add it to Social the same way you did here.